Firestore/core/src/core/key_field_in_filter.h

```diff
@@ -32,6 +34,11 @@ class KeyFieldInFilter : public FieldFilter {
  public:
   KeyFieldInFilter(model::FieldPath field, model::FieldValue value);
 
+  static bool Contains(const model::FieldValue::Array& array_value,
```

Comment:
One way to document that these helpers exist specifically to share code with another class is to make them private and then make that other class a `friend` of this one.